WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasSnippetButton.lua

```diff
@@ -0,0 +1,229 @@
+--[[-----------------------------------------------------------------------------
+SnippetButton Widget, based on AceGUI Button (and WA ToolbarButton)
+Graphical Button.
+-------------------------------------------------------------------------------]]
+local Type, Version = "WeakAurasSnippetButton", 3
```

Comment:
start at 1 maybe? :)

Reply:
Thought that was the AceGUI version. Guess not >.<

Reply:
Heh nope, each AceGui widget has an internal version that has to be increased every time something changes so it can properly upgrade.